web/style/transvision.css

```diff
@@ -782,6 +790,83 @@ input[type="checkbox"]:disabled + label {
     position: relative;
 }
 
+#tooltip_search {
+    display: block;
+    width: 18px;
+    height: 26px;
+    float: left;
+    margin-left: 4px;
+    background: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256" width="18" height="18"><path style="fill: rgb(0,149,221)" d="M128 22.158a105.84 105.84 0 0 0-105.84 105.84A105.84 105.84 0 0 0 128 233.838a105.84 105.84 0 0 0 105.84-105.84A105.84 105.84 0 0 0 128 22.158zm0 32.76c5.16.117 9.55 1.875 13.18 5.273 3.34 3.576 5.07 7.94 5.19 13.097-.12 5.156-1.85 9.404-5.19 12.744-3.63 3.75-8.02 5.626-13.18 5.626s-9.4-1.875-12.74-5.625c-3.75-3.34-5.63-7.587-5.63-12.743s1.88-9.52 5.63-13.096c3.34-3.397 7.58-5.155 12.74-5.272zm-16.35 53.792h32.79v92.37h-32.79v-92.37z"/></svg>') center no-repeat transparent;
+}
+
+#tooltip_search_text {
+    position: absolute;
+    padding: 8px;
+    top: 25px;
+    right: 50px;
+    width: 450px;
+    font-size: 0.85em;
+    text-align: left;
+}
+
+@media only screen and (max-width: 900px) {
+    #tooltip_search_text {
+        left: -120px;
+        width: 400px;
+    }
+}
+
+@media only screen and (max-width: 640px) {
+    #tooltip_search {
+        height: 20px;
+    }
+
+    #tooltip_search_text {
+        left: -80px;
+        width: 300px;
+    }
+}
+
+@media only screen and (max-width: 480px) {
+    #tooltip_search_text {
+        left: -10px;
+        width: 280px;
+    }
+}
+
+#tooltip_search_close {
+    display: block;
+    position: absolute;
+    right: 8px;
+    top: 8px;
+    font-family: 'Fira Sans UltraLight', Arial, sans-serif;
+    font-size: 25px;
+    line-height: 0.5em;
+    color: #fff;
```

Comment:
I think we want to add `text-decoration: none;` here to remove the underline from the link on hover